lib/grammers-client/src/lib.rs

```diff
@@ -46,3 +46,9 @@ pub(crate) mod utils;
 
 pub use client::{Client, Config, InitParams, SignInError};
 pub use types::{button, reply_markup, ChatMap, InputMessage, Update};
+
+pub use grammers_crypto;
+pub use grammers_mtproto;
+pub use grammers_mtsender;
```

Comment:
I'd also prefer to re-export only what's needed here (so errors and reconnection policy)